spec/features/01_user_adds_a_new_piece_of_music_spec.rb

```diff
@@ -0,0 +1,32 @@
+require "rails_helper"
+require "spec_helper"
+
+feature "users can add a new piece of music" do
+  scenario "user adds new piece of music successfully" do
+
+    visit new_piece_path
+    expect(page).to have_content "Add New Music"
+    dummy_data
```

Comment:
Now that this is not in the same file name, consider adding a more usefulk method name that helps explain what it is doing!